### DISCUSSION TITLE
Directus static folder

### DISCUSSION THREAD

**Discussion by neosa** [directus/directus#16795] [Feature Requests]
Good evening, I want to reload all the images from the Wordpress upload folder into a static folder and get a link to the image in the text field to display on the site.

At the moment I see the ability to upload all images via the upload form, but if there are more than 10,000 images, it seems to me the fastest option would be to create a static visible folder and just put all the images there

I did not find any information about this in the manual, as well as in the env file

**Comment by paescuj** [comment]
Not sure what the best solution would be for your use case... But if you want to upload the images to a static folder and have them served via Directus, you should be able to register this folder via an [extension](https://docs.directus.io/extensions/introduction.html) and map it to a route (e.g. `/images`).
The extension might look something like this (not tested):
```js
export default ({ init }) => {
  init("app.before", ({ app }) => {
    app.use('/images', express.static('<path to your folder>'));
  });
};
```

**Reply by neosa** [reply]
I also looked in the direction of this approach, but of course it would be nice to register a folder in the env static folder out of the box :)

**Reply by br41nslug** [reply]
Using this approach the images will indeed be hosted under `/images` together with Directus however Directus will not be aware of these files (they wont be in the file library, linkable to collections nor transformable using the `/assets` endpoint)

**Reply by neosa** [reply]
I don't need Directus to know about them. This is only needed for migrating from Wordpress. Now some of my new pages use images with Directus, some use links through a text field with wordpress

**Reply by neosa** [reply]
And I want to not lose all the images, just put them in a static folder and delete this wordpress already

**Reply by neosa** [reply]
Directus, as I understand it, adds it to the database while loading images and then pulls up the desired image in the field by ID. Directus also renames files during download. And it will be very expensive for me to transfer images from Wordpress when images are sewn up in articles

**Reply by br41nslug** [reply]
Ah yeah, then i'd say what paescuj suggests if you really need it integrated with the directus installation if not i'd personally probably host those separately on another subdomain or even a CDN.

**Reply by neosa** [reply]
Placement is not a problem, but then the subdomain changes, therefore SEO is lost

**Reply by neosa** [reply]
therefore, during the move, I would like to minimize links to images and reconnect the subdomain from wordpress to directus in this case, all links to images will be identical
